packages/truffle-compile/scripts/test.sh

```diff
@@ -3,7 +3,7 @@
 set -o errexit
 
 if [ "$CI" = true ]; then
-  mocha --timeout 10000 $@
+  mocha ./test/** ./test/**/* --timeout 10000 $@
 else
-  mocha --invert --grep native --timeout 10000 $@
+  mocha ./test/** ./test/**/* --invert --grep native --timeout 10000 $@
```

Comment:
Why remove those args?

Reply:
Did you mean "why not remove those args"? If that was your question, mocha doesn't seem to dig down far enough in the test directories without those.